lighthouse-core/audits/audit.js

```diff
@@ -155,14 +158,12 @@ class Audit {
       throw new Error('generateAuditResult requires a rawValue');
     }
 
-    // eslint-disable-next-line prefer-const
     let {score, scoreDisplayMode} = Audit._normalizeAuditScore(audit, result);
 
     // If the audit was determined to not apply to the page, we'll reset it as informative only
-    let informative = audit.meta.informative;
     if (result.notApplicable) {
       score = 1;
```

Comment:
should we be enforcing the other things we say we do? Like no score if not binary or numeric display mode?

Reply:
yeah we're gonna do that in followup